cmake/libre-config.cmake

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/libre.cmake")
```

Comment:
What is the difference between these two files:

```
libre-config.cmake
re-config.cmake
```

could they be the same file ?